### ISSUE TITLE
Crossed out tasks keep appearing

### ISSUE THREAD

**Issue by tijlleenders** [tijlleenders/ZinZen#1341]
**Describe the bug**
Crossed out Tasks keep appearing (as crossed out) today even though I completed them yesterday and there are no more hours left in the goal.

**Comment by Tushar-4781** [comment]
this is no bug as the hrs spent strategy is not full ready for normal use, as per our discussion.

**Comment by Tushar-4781** [comment]
Blocked by #1324

**Comment by tijlleenders** [comment]
Blocked by #1407

**Comment by tijlleenders** [comment]
> Blocked by #1324

Don't think that's relevant anymore.

**Comment by tijlleenders** [comment]
Duplicate of #1137
